src/components/add-todo.jsx

```diff
@@ -21,11 +21,69 @@ class AddTodo extends React.Component {
     }
   }
 
+  componentDidMount() {
+    this.textInput.focus();
+  }
+
   render () {
     return (
-      <form onSubmit={this.handleSubmit.bind(this)}>
-        <input type="text" value={this.state.value} onChange={this.handleChange.bind(this)}/>
-        <button type="submit">Add todo</button>
+      <form onSubmit={this.handleSubmit.bind(this)} className="add-todo">
+        <input
+          type="text"
+          value={this.state.value}
+          placeholder="New todo..."
+          ref={(input) => { this.textInput = input; }}
```

Comment:
I think you can set the HTML `autofocus` property on the input field instead of using the component lifecycle hook.